tests/test_update.py

```diff
@@ -612,6 +612,12 @@ def test_signed_updates(
         """Test various combinations of signed and unsigned, present and non-
         present verification keys."""
 
+        if (
+            sig_case.label
+            == "EC, Correctly signed, but header does not match checksum, key present"
+        ):
+            pytest.skip("MEN-6845")
```

Comment:
`skipif`?

Reply:
I think I'll leave it simply because it's similar to the other one. It will go away pretty soon anyway.